domchanger.js

```diff
@@ -347,6 +346,9 @@ function processTag(array) {
   else {
     body = array.slice(1);
   }
+  if (Array.isArray(body[0]) && !body[0].length) {
+    body[0] = "";
```

Comment:
thinking about it more, it actually doesnt make sense to remove empty nodes as they can be used just for styling or layout, eg `.myFoo:after { content: "hi"; }`. some of these style rules may not be inline, so it's not even sufficient to check and ensure that removing them outright would have no side-effects. it should be up to the author to have explicit removal be conditional within the returned `render`.